<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddUserDialog</class>
 <widget class="QDialog" name="AddUserDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>480</width>
    <height>520</height>
   </rect>
  </property>
  <property name="styleSheet">
   <string notr="true">
    QDialog { background-color: transparent; }
    QFrame#dialog_content {
        background-color: white;
        border-radius: 16px;
        border: 1px solid #E2E8F0;
    }
    QLabel {
        font-family: "Segoe UI";
        color: #334155;
        font-weight: bold;
        font-size: 14px;
    }
    QLineEdit, QComboBox {
        background-color: #F8FAFC;
        border: 1px solid #E2E8F0;
        border-radius: 10px;
        padding: 8px 12px;
        font-size: 14px;
        color: #0F172A;
        min-height: 25px;
    }
    QLineEdit:focus, QComboBox:focus {
        border: 2px solid #06B6D4;
        background-color: white;
    }
    QComboBox::drop-down { border: none; width: 20px; }
   </string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="leftMargin"><number>20</number></property>
   <property name="topMargin"><number>20</number></property>
   <property name="rightMargin"><number>20</number></property>
   <property name="bottomMargin"><number>20</number></property>

   <item>
    <widget class="QFrame" name="dialog_content">
     <property name="frameShape"><enum>QFrame::StyledPanel</enum></property>
     <property name="frameShadow"><enum>QFrame::Raised</enum></property>

     <layout class="QVBoxLayout" name="layout_content">
      <property name="spacing"><number>24</number></property>
      <property name="leftMargin"><number>32</number></property>
      <property name="topMargin"><number>32</number></property>
      <property name="rightMargin"><number>32</number></property>
      <property name="bottomMargin"><number>32</number></property>

      <item>
       <widget class="QLabel" name="label_header">
        <property name="text"><string>Add New User</string></property>
        <property name="styleSheet"><string>font-size: 24px; font-weight: 800; color: #0F172A; margin-bottom: 5px;</string></property>
       </widget>
      </item>

      <item>
       <layout class="QVBoxLayout" name="form_layout">
        <property name="spacing"><number>20</number></property>

        <item>
         <layout class="QVBoxLayout" name="vbox_name">
          <property name="spacing"><number>8</number></property>
          <item><widget class="QLabel" name="lbl_name"><property name="text"><string>Staff Name</string></property></widget></item>
          <item><widget class="QLineEdit" name="input_name"><property name="placeholderText"><string>e.g. John Doe</string></property></widget></item>
         </layout>
        </item>

        <item>
         <layout class="QVBoxLayout" name="vbox_role">
          <property name="spacing"><number>8</number></property>
          <item><widget class="QLabel" name="lbl_role"><property name="text"><string>Role</string></property></widget></item>
          <item>
           <widget class="QComboBox" name="input_role">
            <item><property name="text"><string>Cashier</string></property></item>
            <item><property name="text"><string>Manager</string></property></item>
            <item><property name="text"><string>Stock Clerk</string></property></item>
           </widget>
          </item>
         </layout>
        </item>

        <item>
         <layout class="QVBoxLayout" name="vbox_pass">
          <property name="spacing"><number>8</number></property>
          <item><widget class="QLabel" name="lbl_pass"><property name="text"><string>Password</string></property></widget></item>
          <item>
           <widget class="QLineEdit" name="input_password">
            <property name="echoMode"><enum>QLineEdit::Password</enum></property>
            <property name="placeholderText"><string>Set initial password</string></property>
           </widget>
          </item>
         </layout>
        </item>

       </layout>
      </item>

      <item><spacer name="verticalSpacer"><property name="orientation"><enum>Qt::Vertical</enum></property><property name="sizeHint" stdset="0"><size><width>20</width><height>40</height></size></property></spacer></item>

      <item>
       <layout class="QHBoxLayout" name="actions_layout">
        <property name="spacing"><number>12</number></property>
        <item><spacer name="hspacer"><property name="orientation"><enum>Qt::Horizontal</enum></property><property name="sizeHint" stdset="0"><size><width>40</width><height>20</height></size></property></spacer></item>

        <item>
         <widget class="QPushButton" name="btn_cancel">
          <property name="text"><string>Cancel</string></property>
          <property name="cursor"><cursorShape>PointingHandCursor</cursorShape></property>
          <property name="styleSheet">
           <string notr="true">
            QPushButton {
               background-color: transparent;
               color: #475569;
               font-weight: bold;
               font-size: 14px;
               padding: 10px 20px;
               border-radius: 10px;
            }
            QPushButton:hover { background-color: #F1F5F9; }
           </string>
          </property>
         </widget>
        </item>

        <item>
         <widget class="QPushButton" name="btn_create">
          <property name="text"><string>Create Account</string></property>
          <property name="cursor"><cursorShape>PointingHandCursor</cursorShape></property>
          <property name="styleSheet">
           <string notr="true">
            QPushButton {
               background-color: #0891B2;
               color: white;
               font-weight: bold;
               font-size: 14px;
               padding: 10px 24px;
               border-radius: 10px;
            }
            QPushButton:hover { background-color: #0E7490; }
           </string>
          </property>
         </widget>
        </item>
       </layout>
      </item>

     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>btn_cancel</sender>
   <signal>clicked()</signal>
   <receiver>AddUserDialog</receiver>
   <slot>reject()</slot>
  </connection>
  <connection>
   <sender>btn_create</sender>
   <signal>clicked()</signal>
   <receiver>AddUserDialog</receiver>
   <slot>accept()</slot>
  </connection>
 </connections>
</ui>